lib/game.rb

```diff
@@ -1,8 +1,9 @@
 class Game
-  def initialize(board, checker, player1, player2)
+  def initialize(board, checker, player1, player2, you, other)
```

Comment:
the you/other dichotomy is confusing to me, it feels like it's another representation of the player1/player2 concept, there should be a way to merge the ideas